jerry-core/debugger/debugger-ws.h

```diff
@@ -64,6 +64,23 @@ typedef struct
 } jerry_debugger_send_header_t;
 
 /**
+ * Incoming message: next message of string data.
+ */
+typedef struct
+{
+  uint8_t type; /**< type of the message */
+} jerry_debugger_receive_string_data_part_t;
+
+/**
+ * String data for eval and client source.
+ */
+typedef struct
+{
+  uint32_t string_size; /**< total size of the client source */
+  uint32_t string_offset; /**< current offset in the client source */
+} jerry_debugger_string_data_t;
```

Comment:
The message type is in the jerry_debugger_string_data_part_t struct. This solution was in the eval data struct, so i did not changed it.